tests/Futures/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ target_link_libraries(arango_tests_futures
   PRIVATE
     arango_futures
     gtest
+    boost_boost
```

Comment:
why?

Reply:
Good question! I don't remember why I added this - but it can probably be removed now.